GOVERNANCE.md

```diff
@@ -189,10 +214,31 @@ Example of list of contributions:
   organization
 * Other participation in the wider Node.js community
 
-The nomination passes if no collaborators oppose it after one week, and if the
-nominee publicly accepts it. In the case
-of an objection, the TSC is responsible for working with the individuals
-involved and finding a resolution.
+The nomination passes if no collaborators _explicitly_ oppose it after one week.
+In the case of an objection, the TSC is responsible for working with the
+individuals involved and finding a resolution. The TSC may, following
+typical TSC consensus seeking processes, choose to advance a nomination that
+has otherwise failed to reach a natural consensus or clear path forward even
+if there are outstanding objections.
+
+Explicit opposition would typically be signaled as some form of clear
+and unambiguous comment like, "I don't believe this nomination should pass".
+These _should_ be paired with clear suggestions for positive, concrete,
+and unambiguous next steps that the nominee can take to overcome the objection
+and allow it to pass. While such suggestions are technically optional, they are
+_strongly encouraged_ to prevent the nomination from stalling indefinitely or
+objections from being overridden by the TSC.
+
+An important rule of thumb is that the nomination process is intended to be
+biased strongly towards implicit approval of the nomination. This means
+discussion and review around the proposal should ideally be more geared
+towards "I have reasons to say no..." as opposed to "Give me reasons to say
+yes...".
+
+Refrain from discussing or debating aspects of the nomination process
+itself directly within a nomination private discussion or public issue.
+Such discussions can derail and frustrate the nomination and cause unnecessary
+friction. Move such discussions to a separate issue or discussion thread.
```

Comment:
Should this be moved to `How to review a collaborator nomination` section? Or maybe we need a new section `How to oppose to a collaborator nomination`